panacea/query_client.go

```diff
@@ -195,14 +192,30 @@ func newTMLogger(conf *config.Config) tmlog.Logger {
 	return logger
 }
 
-func (q verifiedQueryClient) safeUpdateLightClient(ctx context.Context) (*tmtypes.LightBlock, error) {
+// startSchedulingLastBlockCaching updates the LightClient with the last block information and stores the height of this block.
+func (q *verifiedQueryClient) startSchedulingLastBlockCaching() {
+	for {
+		block, err := q.lightClient.Update(context.Background(), time.Now())
+		if err != nil {
+			log.Errorf("failed to refresh last block. %v", err)
+			return
+		}
+		if block != nil {
+			log.Debugf("Refresh last block. Height(%d)", block.Height)
+			q.cachedLastBlockHeight = block.Height
+		}
+		time.Sleep(refreshIntervalTime)
+	}
+}
+
+func (q *verifiedQueryClient) safeUpdateLightClient(ctx context.Context) (*tmtypes.LightBlock, error) {
```

Comment:
This function can be removed as well.

Reply:
It is using here :)
https://github.com/medibloc/panacea-oracle/blob/v0.0.1-alpha.1/panacea/query_client.go#L501